docs/configuration.rst

```diff
@@ -116,6 +116,42 @@ Define a secure connection to an Elastic Cloud cluster::
     datastore.user = rally
     datastore.password = the-password-to-your-cluster
 
+storage
+~~~~~~~
+
+This section defines how client is configured to transfer big files (I.E. track files) from
+remote servers. Available options are:
+
+* ``storage.adapters`` is a comma-separated list of storage adapter implementations specified using the following
```

Comment:
Might want to have a section of `Available Adapters` here. I know it's only `HTTPAdapter` now, but when this does grow, it'd be nice to have.